operator/Makefile

```
@@ -65,7 +65,6 @@ docker-push:
 # find or download controller-gen
 # download controller-gen if necessary
 controller-gen:
```

Since you're always building it now, please make it a proper target:
```suggestion
$(GOBIN)/controller-gen:
```

This way `make` can keep track of it better and only rebuild if necessary.

Please also update the comment above; it's no longer `find or download`.